packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -2,15 +2,41 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:math' as math;
+
 import 'package:flutter/foundation.dart';
 import 'package:flutter/rendering.dart';
 
 import 'framework.dart';
 import 'overscroll_indicator.dart';
+import 'scroll_context.dart';
 import 'scroll_physics.dart';
 
 const Color _kDefaultGlowColor = const Color(0xFFFFFFFF);
 
+// Methodology:
+// 1- Use https://github.com/flutter/scroll_overlay to test with Flutter and
+//    platform scroll views superimposed.
+// 2- Record incoming speed and make rapid flings in the test app.
+// 3- If the scrollables stopped overlapping at any moment, adjust the desired
+//    output value of this function at that input speed.
+// 4- Feed new input/output set into a power curve fitter. Change function
+//    and repeat from 2.
+// 5- Repeat from 2 with medium and slow flings.
+/// Momentum build-up function that mimics iOS's scroll speed increase with repeated flings.
+///
+/// Note the velocity of the last fling is not an important factor. Existing speed
+/// and (related) time since last fling are.
```

Comment:
nit: grammar: the second sentence here ends with "are" but doesn't have an object.